<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="wd_main">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="wd_login">
     <attribute name="title">
      <string>登录</string>
     </attribute>
     <widget class="QPushButton" name="pb_login">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_login_clear">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_login_phone">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_login_pass">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lb_login_phone">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>105</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> 手机号</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_login_pass">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wd_register">
     <attribute name="title">
      <string>注册</string>
     </attribute>
     <widget class="QPushButton" name="pb_register">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_clear">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_reg_user">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_rig_user">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>   用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_reg_phone">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> 手机号</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_reg_pass">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_reg_phone">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_reg_pass">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_food">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 美食</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_life">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string> 生活</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_cartoon">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string> 动漫</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_game">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string> 游戏</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_music">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string> 音乐</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>   喜好  推荐</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_reg_duan">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_sendduan">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_reg_duanxin">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>  短信验证</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
